<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinimumSize</enum>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Contacts</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listContacts">
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinAndMaxSize</enum>
          </property>
          <item>
           <widget class="QPushButton" name="btn_delete">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveFileButton">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Save File as ...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="loadContactsButton">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Load .txt</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="searchContact">
            <property name="text">
             <string>Search Contact</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="searchContact_text"/>
          </item>
          <item row="1" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="firstnameLabel">
            <property name="text">
             <string>Firstname</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="firstname_textEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lastnameLabel">
            <property name="text">
             <string>Lastname</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lastname_textEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="emailLabel">
            <property name="text">
             <string>Email</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="email_textEdit"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="numberPhoneLabel">
            <property name="text">
             <string>Number Phone</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="numberPhone_textEdit"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="commentaryLabel">
            <property name="text">
             <string>Commentaries</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QTextEdit" name="commentary_textEdit"/>
          </item>
          <item row="7" column="1">
           <widget class="QPushButton" name="addContact">
            <property name="text">
             <string>Add Contact</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QPushButton" name="submit">
            <property name="text">
             <string>Save New Contact</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QPushButton" name="update">
            <property name="text">
             <string>Update a Contact</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QPushButton" name="Load">
            <property name="text">
             <string>Open a Data base</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1090</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
